python/tk_framework_adobe/rpc/communicator.py

```diff
@@ -46,6 +46,22 @@
 except ImportError:
     from tank_vendor import six as sgutils
 
+if os.environ.get("SGTK_ENFORE_PROXY_LOCALHOST", "0").strip().lower() not in [
```

Comment:
On second glance, I see a typo here, should be "enforce" instead of "enfore" I assume?